Gruntfile.js

```diff
@@ -23,22 +22,10 @@ module.exports = function(grunt) {
       production: 'publish',
       bamboo: 'bamboo',
 
-      // webrtc-adapter submodule
-      webrtcAdapterSubmodule: 'third_party/adapter/',
-      googleAdapterPath: '<%= webrtcAdapterSubmodule %>/out/adapter.js',
-
       pluginInfoRoot: grunt.option('pluginInfoRoot') || '<%= source %>',
       pluginInfoFile: grunt.option('pluginInfoFile') || 'pluginInfo.js',
 
-      githash: {
-        submodule: {
-          options: {
-            dir: '<%= webrtcAdapterSubmodule %>'
-          }
-        },
-      },
-
-      version: '<%= pkg.version %>-<%= githash.submodule.short %>',
+      version: '<%= pkg.version %>',
```

Comment:
Well, we would no longer be depending on a git-hash, but rather a node dep' version...
If anything, then what we should put next to our version number would be the webrtc-adapter version number.

I'm not sure if that's worth it though...